docs/DatabaseCatalogAPI.yaml

```diff
@@ -643,29 +635,51 @@ components:
     provider:
       name: provider
       in: query
-      description: Filter feeds by provider name using the LIKE operation. Phrase is matched case insensitively to actual provider names.
+      description: List only feeds with the specified value. Can be a partial match. Case insensitive.
       required: false
       schema:
         type: string
-        example: London Transit
+        example: Los Angeles Department of Transportation
+
+    # A provider specific to gtfs realtime simply to have a different example
```

Comment:
I think we should not add  _"extra"_ parameters just for the purpose of having a different example in the SwaggerUI. SwaggerUI users should not be the main consumers of the schema.